cdist/exec/local.py

```diff
@@ -224,8 +224,16 @@ def save_cache(self):
                 shutil.rmtree(destination)
         except PermissionError as e:
             raise cdist.Error("Cannot delete old cache %s: %s" % (destination, e))
-
-        shutil.move(self.base_path, destination)
```

Comment:
Python 3.5 introduced copy_function argument to shutil.move that could be used for this functionality, but Python 3.2 that is minimal python version does not have it so I am using copytree and rmtree.